nav2_msgs/action/FollowPath.action

```diff
@@ -24,4 +24,6 @@ string error_msg
 ---
 #feedback definition
 float32 distance_to_goal
+uint64 closest_path_pose_idx   # index of the pose on global path closest to current robot pose
```

Comment:
I think these should be removed or at least adjusted. The `distance_to_path` is not very accurate since it is the nearest path point, not the actual distance from the path. When using very dense paths, this might be reasonably accurate, but when the path markers are further away, then these measures are pretty poor. 

How would these two values be used by client applications?

Reply:
true, but I think that's the best estimation we have without linearly interpolating the path, which introduces a different assumption. Shouldn't most paths be quite dense if they are coming from a nav2 planner?

For my use case: I am using them to monitor whether the robot has deviated the too much from the path and switch to another behavior. For example: while following a coverage path re-plan a detour path, or requesting user input if the deviation is way too big.

Reply:
removed the extra feedback.
I am not sure I have time to work on the extra util right now, should I open an issue so it is not forgotten?